src/ray/rpc/server_call.h

```diff
@@ -129,13 +128,9 @@ class ServerCallImpl : public ServerCall {
 
   void HandleRequestImpl() {
     state_ = ServerCallState::PROCESSING;
-    (service_handler_.*handle_request_function_)(request_, &reply_,
-                                                 [this](Status status) {
-                                                   // When the handler is done with the
-                                                   // request, tell gRPC to finish this
-                                                   // request.
-                                                   Finish(status);
-                                                 });
+    auto status = (service_handler_.*handle_request_function_)(request_, &reply_);
+    // When the handler is done with the request, tell gRPC to finish this request.
+    Finish(status);
```

Comment:
The `done_callback` is used to allow the request handler to handle a request asynchronously. 
I think it's still needed, e.g., by `WaitRequst` and `FetchOrReconstructObjects`.

Reply:
yes... I forgot that one.